content/medo_manual/werken_met_deelnemers.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Werken met deelnemers
+sort: 300
+---
+
+* TOC
+{:toc}
+
```

Comment:
Lijkt me nuttig om nog een korte inleiding toe te voegen over wat je in Medo kunt doen (persoonlijke gegevens beheren), en wat je in RoQua kunt doen (vragenlijsten (laten) invullen)